# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/cookiestore/moz.build
# Commit: cc3c39aa8fb8
# Full Hash: cc3c39aa8fb8124cca265e84f71a29cbd218b704
# Author: Andrea Marchesini <amarchesini@mozilla.com>
# Date: 2024-09-12 09:23:07
# Regressor Bug: 1475599
# File Overlap Count: 1
# Description:
#   Bug 1475599 - part 2 - CookieStore API - IPDL, r=edgul,webidl,smaug
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D215145
# ==============================================================================

diff -r 2fc9a3d0ad8b -r cc3c39aa8fb8 dom/cookiestore/moz.build
--- a/dom/cookiestore/moz.build	Wed Sep 11 16:26:54 2024 +0000
+++ b/dom/cookiestore/moz.build	Wed Sep 11 16:26:55 2024 +0000
@@ -8,11 +8,16 @@
 EXPORTS.mozilla.dom += [
     "CookieChangeEvent.h",
     "CookieStore.h",
+    "CookieStoreChild.h",
+    "CookieStoreParent.h",
 ]
 
 UNIFIED_SOURCES += [
     "CookieChangeEvent.cpp",
     "CookieStore.cpp",
+    "CookieStoreChild.cpp",
+    "CookieStoreNotifier.cpp",
+    "CookieStoreParent.cpp",
 ]
 
 LOCAL_INCLUDES += [
@@ -20,6 +25,10 @@
     "../events",
 ]
 
+IPDL_SOURCES += [
+    "PCookieStore.ipdl",
+]
+
 include("/ipc/chromium/chromium-config.mozbuild")
 
 FINAL_LIBRARY = "xul"